test/crd.go

```diff
@@ -102,6 +102,7 @@ func Configuration(namespace string, names ResourceNames, imagePath string) *v1a
 					Container: corev1.Container{
 						Image: imagePath,
 					},
+					ContainerConcurrency: 10,
```

Comment:
Can we set ContainerConcurrency explicitly in `e2e/autoscale_test.go`?

It helps with understanding that the # of qps (burst of traffic) and this value are two knobs that'll have an impact on the test. 

Also setting it here has the side effect of impacting other tests.

Reply:
I'm fine with the suggested change as a separate PR - I'd rather make our prow integration tests  stable.